File: 01.JAVA/pg-166-169/pg_166_13.java
import java.util.Scanner;

public class pg_166_13
{
    public static void main(String[]arg)
    {
        Scanner enter = new Scanner(System.in);
        int num1, num2;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("READ TWO TWO-DIGIT INTEGERS AND DETERMINE IF THE SUM OF BOTH INTEGERS PRODUCE AN EVEN NUMBER");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("");

        System.out.print("Write the first integer: ");
        num1 = enter.nextInt();

        System.out.print("Write the second integer:");
        num2 = enter.nextInt();

        if(num1 < 0)
        {
            num1 *= -1;
        }
        if(num2 < 0)
        {
            num2 *= -1;
        }

        int sum = num1 + num2;

        if(sum % 2  == 0)
        {
            System.out.println("THE SUM OF BOTH INTEGERS (" + sum + ") PRODUCES AN EVEN NUMBER.");
        }
        else
            {
                System.out.println("THE SUM OF BOTH INTEGERS (" + sum + ") DOESN'T PRODUCE AN EVEN NUMBER.");
            }
    }
}

File: 03.JAVASCRIPT/pg-166-169/pg_166_13.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJ");
console.log("READ TWO TWO-DIGIT INTEGERS AND DETERMINE IF THE SUM OF BOTH INTERGERS PRODUCE AN EVEN NUMBER");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJ");
console.log("");

let numInput1 = document.querySelector("#num1");
let numInput2 = document.querySelector("#num2");
let btnProof = document.querySelector("#btnProof");
let textar = document.querySelector("#textar");

function validateNumber()
{
    let num1 = Number(numInput1.value);
    let num2 = Number(numInput2.value);

    let sum = num1 + num2;

    if((num1 >= 10 && num1 <= 99) && (num2 >= 10 && num2 <=99))
    {
        if(sum % 2 == 0)
        {
            textar.value = "The sum of the both integers (" + sum + ") is an even number";
        }
        else
        {
            textar.value = "The sum of the both integers (" + sum + ") isn't an even number";
        }
    }
    else
        {
            textar.value = "Some of the written numbers doesn't have two digits. Please try again!";
        }
}

btnProof.addEventListener("click", validateNumber);
numInput1.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});

numInput2.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});